src/tools/devtools/overview.md

```diff
@@ -16,6 +16,16 @@ the following deep dive and use case walkthrough:
 <iframe width="560" height="315" src="{{site.yt.embed}}/_EYk-E29edo" title="Dive in to Flutter DevTools" {{site.yt.set}}></iframe>
 [Dive in to DevTools]({{site.yt.watch}}?v=_EYk-E29edo)
 
+## How to start DevTools?
+
+If your app is started using VSCode GUI, use one of the following ways:    
+1. Click button 'DevTools' on Flutter panel in VS Code
+2. Select the menu `{} > Dart DevTools` at the bottom of VS Code
+
+If your app is started with CLI:
+1. Start DevTools with `dart devtools`
+2. Connect to the app by pasting the app VM Service URI from the app's console
```

Comment:
Users shouldn't have to do this. They should click the DevTools link that is printed to CLI output.